src/main/java/com/example/fooddelivery/controller/CreateOrderController.java

```diff
@@ -25,25 +29,56 @@
     private static final String ADDRESS_FORM = "addressForm";
     private static final String RESTAURANTS = "restaurants";
     private static final String ADDED_PRODUCTS = "addedProducts";
+    private static final String ORDER_ID = "orderId";
+    private static final String ORDERS_LIST = "ordersList";
+    private static final String ORDER_DETAILS = "orderDetails";
     private static final String SESSION_PRODUCTS = "sessionProducts";
     private static final String SESSION_ADDRESS = "sessionAddress";
     private static final String REDIRECT = "redirect:/";
 
     @Autowired
-    private DefaultRestaurantService defaultRestaurantService;
+    private RestaurantService restaurantService;
 
     @Autowired
-    private DefaultCartFacade defaultCartFacade;
+    private CartFacade cartFacade;
+
+    @Autowired
+    private OrderService orderService;
+
+    @Autowired
+    private OrderFacade orderFacade;
 
     @GetMapping()
     public String createOrder(Model model) {
         model.addAttribute(ADDRESS_FORM, new AddressForm());
         model.addAttribute(ADDED_PRODUCTS, new ArrayList<ProductWithQuantityDto>());
-        model.addAttribute(RESTAURANTS, defaultRestaurantService.getRestaurantsNameAndId());
+        model.addAttribute(RESTAURANTS, restaurantService.getRestaurantsNameAndId());
```

Comment:
Why you used service directly in controller? Can you move the call inside restaurant facade?

Reply:
The same problem with facade/service. Let's find some useful links.